lib/net/ssh.rb

```diff
@@ -103,6 +103,7 @@ module SSH
     #
     # This method accepts the following options (all are optional):
     #
+    # * :add_keys_to_agent => yes, no, confirm
```

Comment:
i assume this would be true/false, :confirm, the code bellow does not supports the `:confirm` option does it?